index.d.ts

```diff
@@ -28,21 +32,32 @@ export interface WebViewProps extends React.HTMLAttributes<Electron.WebViewEleme
   onDidGetResponseDetails?: EventListener
   onDidGetRedirectRequest?: EventListener
   onDomReady?: EventListener
-  onPageTitleSet?: EventListener
+  onPageTitleUpdated?: EventListener
```

Comment:
Okay, then we should leave them both in for compatibility. I don't think it's worth bumping up our Electron peer dependency just for this.